astropy/visualization/wcsaxes/ticklabels.py

```diff
@@ -180,6 +180,10 @@ def draw(self, renderer, bboxes, ticklabels_bbox):
 
                     dist = np.hypot(dx, dy)
 
+                    # Avoid RuntimeWarning
+                    if dist == 0 or not np.isfinite(dist):
```

Comment:
It seems OK, but really will have to ask @astrofrog whether this is reasonable. An alternative would be to just change the particular example to not hit this problem.